.github/workflows/matrix.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This workflow is reused, so there is no need to maintain the same step in two different workflows:
- test.yml
- periodic.yml